.github/actions/check-branch-commits/action.yaml

```diff
@@ -0,0 +1,22 @@
+name: "Check Branch Commits"
+description: |
+  Runs the branch commit checker on aptos-core, assuming it is already checked out in the current working directory.
+inputs:
+  TIMEOUT_MINUTES:
+    description: "The number of minutes to wait for branch commit checking to finish."
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Install Dependencies
+      shell: bash
+      run: |
+        echo "${HOME}/bin/" >> $GITHUB_PATH # default INSTALL_DIR to path
+        sudo apt-get update -y && sudo apt-get install -y expect
+    - name: Run branch commit checker
+      shell: bash
+      run: |
+        source "$HOME/.cargo/env" # Required for allowing python access to cargo
+        pip install Levenshtein # Required for Levenshtein distance
```

Comment:
can you please add this dependency to a `requirements.txt` and then install it via pip install -r requirements.txt. Makes it a bit easier to repro this locally.

Reply:
Done. Also did the same for `fullnode-sync`).